apps/api/src/app/integrations/usecases/create-integration/create-integration.usecase.ts

```diff
@@ -24,7 +25,7 @@ export class CreateIntegration {
         active: command.active,
       });
 
-      if (command.active) {
+      if (command.active && command.channel !== ChannelTypeEnum.DIRECT) {
```

Comment:
I think we should add here PUSH aswell, since push is allowed to have multiple connections to such as FCM and APNS